docs/updates/when-the-world-stops.md

```diff
@@ -0,0 +1,216 @@
+### When the world stops
+
+
+One of the most interesting questions is what happens when the debugger stops? Basically, what happens when the world pauses?
+
+Here is a quick run-through that will help give you some context so that you can dig in and answer your own questions.
+
+There are three characters in the story: the client, server, and engine.
+When the page pauses, the engine pauses the page, tells the server,
+and the server tells the client.
+
+This post is going to focus on the work the server does when the page pauses. The code included has been simplified a bit to remove some of the incidental complexity.
+
+---
+
+The [server][script.js] has several page lifecycle hooks that are called when
+something happens. `onDebuggerStatement` is one of these hooks!
+
+```js
+onDebuggerStatement: function (frame) {
+  return this._pauseAndRespond(frame, { type: "debuggerStatement" });
+}
+```
+
+The [server][script.js]'s main job is to tell the client that the page is paused.
+You can see below, that the `_paused` function is building a `packet`, this packet
+has all of the data the client needs to inform the programmer about the pause state.
+The `_paused` function also does one other interesting thing here as well. It starts,
```

Comment:
`It starts,` could be read with two different tones... maybe `When the function is triggered, it registers ...` -- this makes `it` clear, and doesn't have the weird pause.